src/python/pants/engine/internals/native.py

```diff
@@ -257,6 +257,9 @@ def new_scheduler(
             instance_name=execution_options.remote_instance_name,
             root_ca_certs_path=execution_options.remote_ca_certs_path,
             oauth_bearer_token_path=execution_options.remote_oauth_bearer_token_path,
+            store_headers=tuple(
+                (k, v) for (k, v) in execution_options.remote_store_headers.items()
```

Comment:
do we need this `(k, v) for (k, v)`? Seems redundant to me, but maybe I'm missing something

Reply:
We don't! Bad copypasta.